ca/server.go

```diff
@@ -373,7 +383,16 @@ func (s *Server) Run(ctx context.Context) error {
 			case state.EventUpdateCluster:
 				s.updateCluster(ctx, v.Cluster)
 			}
-
+		case <-ticker.C:
```

Comment:
We could have made this outside of the main select, and an external-CA only functionality.